packages/botonic-core/src/hubtype-service.js

```diff
@@ -19,6 +19,10 @@ const _HUBTYPE_API_URL_ = getWebpackEnvVar(
 const ACTIVITY_TIMEOUT = 20 * 1000 // https://pusher.com/docs/channels/using_channels/connection#activitytimeout-integer-
 const PONG_TIMEOUT = 5 * 1000 // https://pusher.com/docs/channels/using_channels/connection#pongtimeout-integer-
 export class HubtypeService {
+  user
```

Comment:
afaik this is not a common idiom in js. Like in python, IDEs (and developers) understand the idiom of using as class fields those assigned in the constructor. If we migrate ever migrate to typescript, we'll see then what's the best way to declare the fields here.

Reply:
cool, removing it 👍